source/hardware/controllers/korg_kaoss_dj.rst

```diff
@@ -62,29 +62,30 @@ Mapping
 +------------------------------------------------+------------------------------------------------+------------------------------------------------+------------------------------------------------+
 | 12                                             | Touch Slider mode button                       | Switches between the three touch slider mode   |                                                |
 +------------------------------------------------+------------------------------------------------+------------------------------------------------+------------------------------------------------+
-| 13                                             | Touch Slider (Normal mode)                     | Left Side - Nudges (pitch -)                   | Moves to the specified position in the song    |
+| 13                                             | Touch Slider (Normal mode)                     | Left - Nudges (pitch -)                        | Moves to the specified position in the song    |
```

Comment:
this and the rows below:
I think some formatting or line breaks in  columns 3+4 would improve readability a lot.
Current state looks like this:
![image](https://user-images.githubusercontent.com/5934199/133942969-dd93bc58-552f-4b5c-b054-0bb5524827ae.png)

The text can also be improved, for example
```
Set hotcue / go to set hotcue
Left: 1 / Center: 2 / Right: 3
```
and in a similiar way for the shifted mode description.

Reply:
the text is in fact formatted with line-breaks like this:
```
| ... | ... | Left - Deletes Hot Cue 1  |
| ... | ... | Center - Deletes Hot Cue 2|
| ... | ... | Right- Deletes Hot Cue 3  |
```

It should be formatted in the doc (parsed via Sphinx i guess) as 3 individual lines.
However Github's preview interprets this as a Markdown table that does not support newlines which is why it looks like a mess in your screenshot 😄

Reply:
this is not just a preview, it's the actual online manual.
for markdown we need better separators then to increase readability.

Reply:
hmm, well as far as I know this is supposed to be the correct way for specifying newlines in re-structured text... 
I think what's happening here is that the lines exceed the max. number of characters and then auto-wrapping takes over and the newlines are no longer correctly parsed...

this would explain this:  (from the online-doc)  
(all entries are written with newlines but it only works as intended for "Deletes Hot Cue ."
![image](https://user-images.githubusercontent.com/22773387/134030112-3aa8bd26-e1ed-49cf-a106-8812a9829aeb.png)

Reply:
yes, I understand that.
what I was proposing as "better separators" is simply something like `Left: set hotcue... ` instead of `Left - set hotcue... `

Reply:
ok, yep that seems reasonable! 
I'll  try to come up with a better readable version as soon as i find the time!